samples/managed_subscribe/README.md

```diff
@@ -11,18 +11,38 @@ disconnects it will stop publishing to that dynamically generated topic.
 
 ### Setup
 
-1. Create the four config files with the following contents, if they are not already there:<br><br>
----- consumer_settings.yaml ----<br>
-`invehicle_digital_twin_uri: "http://0.0.0.0:5010"`<br><br>
----- invehicle_digital_twin_settings.yaml ----<br>
-`invehicle_digital_twin_authority: "0.0.0.0:5010"`<br><br>
----- managed_subscribe_settings.yaml ----<br>
-`base_authority: "0.0.0.0:5010"`<br>
-`managed_subscribe_uri: "http://0.0.0.0:50051"`<br><br>
----- provider_settings.yaml ----<br>
-`invehicle_digital_twin_uri: "http://0.0.0.0:5010"`<br><br>
-
-1. Build the invehicle_digital_twin service with the `managed_subscribe` feature enabled.
+1. Create the four config files with the following contents, if they are not already there:
+
+    ---- consumer_settings.yaml ----
+
+    ```yaml
+    invehicle_digital_twin_uri: "http://0.0.0.0:5010"
```

Comment:
this is out of scope for this PR, but I think it would be very helpful to have sample configs for these so that users don't have to manually create them. might be worth creating an issue to track

Reply:
We should have a standardized config process for all of our services if possible. I will look into getting an issue created.